### PR TITLE
migrate cmd/kubeadm/app/util/version.go to structured logging

### PR DESCRIPTION
**Pull request by thickforest**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by cndoit18 — COMMENTED** [review]
I don't quite understand the purpose of such a modification.
And you created a number of similar PRs that look like they could be compressed into one PR

File: cmd/kubeadm/app/util/version.go [hunk 1]
```diff
@@ -251,7 +251,7 @@ func validateStableVersion(remoteVersion, clientVersion string) (string, error)
 	if verClient.Major() < verRemote.Major() ||
 		(verClient.Major() == verRemote.Major()) && verClient.Minor() < verRemote.Minor() {
 		estimatedRelease := fmt.Sprintf("stable-%d.%d", verClient.Major(), verClient.Minor())
-		klog.Infof("remote version is much newer: %s; falling back to: %s", remoteVersion, estimatedRelease)
+		klog.InfoS("remote version is much newer, falling back", "remoteVersion", remoteVersion, "backtoVersion", estimatedRelease)
 		return estimatedRelease, nil
```
Comment (wangyysde) [inline cmd/kubeadm/app/util/version.go:255]: As if remoteVersion and estimatedRelease are regular string. So we needn't change this line.

Comment (wangyysde) [reply cmd/kubeadm/app/util/version.go:255]: We should close this PR, i think.

**Comment by wangyysde** [comment]
close

**Pull request closed without merging**